Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
I didn't know if we'd hit issues with this. Locally I get this when I modify the `ap2s` formula:

```
brew install --build-bottle -fs ap2s
...
brew rm ap2s
...
brew install /Users/Dominyk/Downloads/a2ps-4.14.yosemite.bottle.2.tar.gz

Warning: Formula file is modified!
Building from source because Library/Formula/a2ps.rb has local changes
To install from a bottle instead, run with --force-bottle
==> Downloading http://ftpmirror.gnu.org/a2ps/a2ps-4.14.tar.gz
Already downloaded: /usr/local/var/homebrew/cache/a2ps-4.14.tar.gz
==> Downloading https://trac.macports.org/export/56498/trunk/dports/print/a2ps/files/patch-contrib_sample_Makefile.in
Already downloaded: /usr/local/var/homebrew/cache/a2ps--patch-5a34c101feb00cf52199a28b1ea1bca83608cf0a1cb123e6af2d3d8992c6011f.in
==> Downloading https://trac.macports.org/export/56498/trunk/dports/print/a2ps/files/patch-lib__xstrrpl.c
Already downloaded: /usr/local/var/homebrew/cache/a2ps--patch-89fa3c95c329ec326e2e76493471a7a974c673792725059ef121e6f9efb05bf4.c
```

Does the bot handle that automatically?